secretmanager/api/QuickStart/QuickStart.csproj

```diff
@@ -5,6 +5,6 @@
     <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Google.Cloud.SecretManager.V1Beta1" Version="1.0.0-beta01" />
+    <PackageReference Include="Google.Cloud.SecretManager.V1" Version="1.0.0-beta01" />
```

Comment:
I think the `version` may also need to change? I'm not sure what the new package version is though.

Reply:
See below.